README.md

```diff
@@ -146,6 +146,38 @@ The following example will generate Magento 2.2 documentation only.
 
 If you don't have the `_config.local.yml` file at the root of your `devdocs/` directory, the rake will generate all versions of the documentation.
 
+### To build v2.0 documentation
```

Comment:
Other sections start with the action verb Build, maybe keep this Build v2.0 documentation?

Reply:
No, I'd rather keep this heading consistent with the heading the immediately precedes this section.